cli/cmd/init.go

```diff
@@ -0,0 +1,344 @@
+		}
+	}
+
+	if notFoundPluginsErrors != nil {
+		return notFoundPluginsErrors
+	}
+
+	if source == "" {
+		source, err = selectSource(allPlugins, acceptDefaults)
+		if err != nil {
+			return err
+		}
+		source, _ = normalizePluginPath(source)
+	}
+	_, sourceIndex, _ := lo.FindIndexOf(allPlugins, pluginFilter(source, cqapi.PluginKindSource))
+
+	if destination == "" {
+		destination, err = selectDestination(allPlugins, acceptDefaults)
+		if err != nil {
+			return err
+		}
+		destination, _ = normalizePluginPath(destination)
+	}
+	_, destinationIndex, _ := lo.FindIndexOf(allPlugins, pluginFilter(destination, cqapi.PluginKindDestination))
+
+	sourcePlugin := allPlugins[sourceIndex]
+	fmt.Printf("Getting configuration for source plugin %s...\n", bold.Sprintf("%s/%s@%s", sourcePlugin.TeamName, sourcePlugin.Name, *sourcePlugin.LatestVersion))
+	sourceVersion, err := api.GetPluginVersion(apiClient, sourcePlugin.TeamName, sourcePlugin.Kind, sourcePlugin.Name, *sourcePlugin.LatestVersion)
+	if err != nil {
+		return fmt.Errorf("failed to get source plugin %s/%s@%s version %w", sourcePlugin.TeamName, sourcePlugin.Name, *sourcePlugin.LatestVersion, err)
+	}
+
+	destinationPlugin := allPlugins[destinationIndex]
+	fmt.Printf("Getting configuration for destination plugin %s...\n", bold.Sprintf("%s/%s@%s", destinationPlugin.TeamName, destinationPlugin.Name, *destinationPlugin.LatestVersion))
+	destinationVersion, err := api.GetPluginVersion(apiClient, destinationPlugin.TeamName, destinationPlugin.Kind, destinationPlugin.Name, *destinationPlugin.LatestVersion)
+	if err != nil {
+		return fmt.Errorf("failed to get destination plugin %s/%s@%s version %w", destinationPlugin.TeamName, destinationPlugin.Name, *destinationPlugin.LatestVersion, err)
+	}
+
+	if specPath == "" {
+		specPath = sourcePlugin.Name + "_to_" + destinationPlugin.Name + ".yaml"
+	}
+	fmt.Printf("Writing spec to %s...\n", bold.Sprint(specPath))
+	var yamlSpec strings.Builder
+	sourceConfig := configForSourcePlugin(sourcePlugin, sourceVersion)
+	yamlSpec.WriteString(strings.ReplaceAll(sourceConfig, "DESTINATION_NAME", destinationPlugin.Name))
+	yamlSpec.WriteString("\n---\n")
+	yamlSpec.WriteString(configForDestinationPlugin(destinationPlugin, destinationVersion))
+
+	if err := os.WriteFile(specPath, []byte(yamlSpec.String()), 0644); err != nil {
+		return fmt.Errorf("failed to write spec file %w", err)
+	}
+
+	if user != nil {
+		successful.Println("Sync spec file generated successfully!")
+		fmt.Println("You can now run the sync with the following command:")
+		bold.Printf("cloudquery sync %s\n", specPath)
+	} else {
+		successful.Println("Sync spec file generated successfully!")
+		fmt.Println("To run the sync please login with the following command:")
+		bold.Println("cloudquery login")
+		fmt.Println("Then you can run the sync with the following command:")
+		bold.Printf("cloudquery sync %s\n", specPath)
+	}
```

Comment:
So assuming that this tool's purpose is to make it as easy as possible for anyone to configure a sync, it would be grand to provide a link to the docs for the chosen plugins right here, because this tool makes you not have to think about how to configure the sync, but you still have to do the auth and that still makes you go to the same page to learn how.

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/19016/commits/38a34dea80ec3b91f9db3e469c3f5dcd8ca2e0f6, also brought up in https://github.com/cloudquery/cloudquery-issues/issues/2312#issuecomment-2311879206